.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - "BAT_HELPER_SERVER=http://localhost:3004"
```

Comment:
I think we can remove helper here as well as in `base.yml` and in `.env.example`

Reply:
good catch